bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -82,13 +82,24 @@ spec:
           - -ec
           - |-
             ID="${MY_POD_NAME#"{{ $fullname }}-"}"
+            {{- if .Values.auth.tls.autoGenerated }}
+            if [[ -f "/certs/keycloak-${ID}.key" ]] && [[ -f "/certs/keycloak-${ID}.crt" ]] && [[ -f "/certs/ca.crt" ]]; then
+                cp "/certs/ca.crt" "/opt/bitnami/keycloak/certs/ca.crt"
+                cp "/certs/keycloak-${ID}.key" "/opt/bitnami/keycloak/certs/keycloak.key"
+                cp "/certs/keycloak-${ID}.crt" "/opt/bitnami/keycloak/certs/keycloak.crt"
```

Comment:
At this point, I think the environment variables will be present. Could we use them instead of hardcoding paths?

Reply:
I adapted this feature to the chart's previous SSL configuration. For better context, check the entire initContainer command:
https://github.com/bitnami/charts/blob/04ef39d75d131cf84f213751ab37cd5ecdc365da/bitnami/keycloak/templates/statefulset.yaml#L80-L102

Keycloak chart requires providing 1 certificate Keystore per node, with every Keystore named "keycloak-${ID}.keystore.jks" where `ID` is the Pod sequence number.

The goal of this initContainer is to move the node cert from the `shared-certs` volume (/certs) into the location where Keycloak will try to find the certs (/opt/bitnami/keycloak/certs). For Kafka, this logic is instead included in the `setup.sh` configmap.

Additionally, I don't think there is a method to extract the Pod index as an environment variable to be used in another environment variable.